src/assets/styles/includes.scss

```diff
@@ -74,3 +74,61 @@
     pointer-events: none;
   }
 }
+
+@mixin xButton() {
+  :host([scale="s"]) {
+    .x-button {
+      inline-size: theme("spacing.4");
+      block-size: theme("spacing.4");
+    }
+  }
+
+  :host([scale="m"]) {
+    .x-button {
+      inline-size: theme("spacing.6");
+      block-size: theme("spacing.6");
+    }
+  }
+
+  :host([scale="l"]) {
+    .x-button {
+      inline-size: theme("spacing.8");
+      block-size: theme("spacing.8");
+    }
+  }
+
+  .x-button {
+    @apply flex
+    self-center
+    focus-base
+    transition-default
+    text-color-3
+    m-0
+    border-2
+    cursor-pointer
+    items-center
+    bg-transparent;
+    -webkit-appearance: none;
+    display: flex;
```

Comment:
flex is already applied using `apply`.

Reply:
I think some other css statements are duplicated as well.

Reply:
Yikes! Can't believe I missed that. 😅 

Thanks for the great catches. Fixing right up!

Reply:
Fixed!